scripts/xtensa-build-all.sh

```diff
@@ -177,7 +177,7 @@ do
 		PATH=$pwd/../$HOST/bin:$OLDPATH
 	fi
 
-	./configure --with-arch=xtensa --with-platform=$PLATFORM --with-root-dir=$ROOT --host=$HOST \
+	./configure --with-arch=xtensa --with-platform=$PLATFORM --with-root-dir=$ROOT --host=$HOST --enable-debug=no\
```

Comment:
Can we pass this into xtensa-build-all.sh as a -d switch.

Reply:
Thas is a good idea